external/google/protobuf/any.proto

```diff
@@ -33,7 +33,7 @@ syntax = "proto3";
 package google.protobuf;
 
 option csharp_namespace = "Google.Protobuf.WellKnownTypes";
-option go_package = "github.com/golang/protobuf/ptypes/any";
+option go_package = "google.golang.org/protobuf/types/known/anypb";
```

Comment:
where do these files come from?  Did you make the change, or are they somewhere from upstream and vendored here (and you just replaced them with newer)?

Reply:
Great question! 

These files come from the protoc download and are part of the protobuf library. 
We use them as part of the generated-code target.
From what I can tell they are managed manually and tied to the version of protoc used. So this is the latest tied to protoc 3.15.8.

Reply:
I have part of the install-protoc target manage that folder now so it stays up to date as that version changes.